.gitignore

```diff
@@ -1,2 +1,3 @@
 # PhpStorm
 .idea
+.DS_Store
```

Comment:
This needs to go in your local .gitingore not in here (probably best to remove .idea too) since this has nothing to do with the project ;)